GitUI/UserControls/RevisionGridClasses/GitRefListsForRevision.cs

```diff
@@ -15,45 +16,52 @@ public GitRefListsForRevision(GitRevision revision)
         {
             _allBranches = revision.Refs.Where(h => !h.IsTag && (h.IsHead || h.IsRemote)).ToArray();
             _localBranches = _allBranches.Where(b => !b.IsRemote).ToArray();
-            _branchesWithNoIdenticalRemotes = _allBranches.Where(
-                b => !b.IsRemote || !_localBranches.Any(lb => lb.TrackingRemote == b.Remote && lb.MergeWith == b.LocalName)).ToArray();
+            _branchesWithNoIdenticalRemotes = _allBranches.Where(b => !b.IsRemote || 
+                                                                      !_localBranches.Any(lb => lb.TrackingRemote == b.Remote && lb.MergeWith == b.LocalName))
+                                                          .ToArray();
 
             _tags = revision.Refs.Where(h => h.IsTag).ToArray();
         }
 
-        public IGitRef[] AllBranches
```

Comment:
Re# - C# 6 style 
Me - `.AsEnumerable`

The reason is as follows:
Readonly Array/List properties which return private fields as-is is generally a bad idea, since the underlying collections aren't made immutable. It can be very easy to corrupt the underlying collection and very difficult to debug. To address this I changed the properties to be enumerable.
Also the way these properties are used, enumerable seems like a better choice anyway.